release-notes/opensearch-security.release-notes-2.6.0.0.md

```diff
@@ -0,0 +1,15 @@
+## 2023-02-28 Version 2.6.0.0
+
+Compatible with OpenSearch 2.6.0
+
```

Comment:
Do we need to add REST admin permissions to the enhancements from this PR https://github.com/opensearch-project/security/pull/2411

Reply:
It hasn't been backported yet. The automatic backport failed and it needs manual backporting. The contributor for the change said he will try manually backporting the change tomorrow.